harness/determined/cli/workspace.py

```diff
@@ -146,6 +148,43 @@ def list_pools(args: argparse.Namespace) -> None:
     )
 
 
+def list_workspace_namespace_bindings(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_ListWorkspaceNamespaceBindings(sess, id=w.id)
+    if resp.namespaceBindings:
+        cluster_namespaces = []
+        for binding in resp.namespaceBindings:
+            cluster_namespaces.append([binding.clusterName, binding.namespaceName])
+        render.tabulate_or_csv(
+            headers=["Cluster", "Namespace"],
+            values=cluster_namespaces,
+            as_csv=False,
+        )
+    else:
+        print("No workspace-namespace bindings for workspace " + args.workspace_name)
+    return None
+
+
+def add_workspace_namespace_binding(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    content = bindings.v1AddWorkspaceNamespaceBindingRequest(
+        id=w.id,
+        clusterName=args.cluster_name,
+        namespaceName=args.namespace_name,
+    )
+    workspace_name = str(args.workspace_name)
+
+    wsns = bindings.post_AddWorkspaceNamespaceBinding(sess, body=content, id=w.id)
+
+    if args.cluster_name:
+        cluster_details = "for cluster " + wsns.cluster_name + "."
```

Comment:
initialize cluster_details before the if statement, otherwise you will receive this err:

> Traceback (most recent call last):
  File "/Users/salonigupta/determined/harness/determined/cli/cli.py", line 274, in main
    parsed_args.func(parsed_args)
  File "/Users/salonigupta/determined/harness/determined/cli/workspace.py", line 184, in add_workspace_namespace_binding
    cluster_details)
UnboundLocalError: local variable 'cluster_details' referenced before assignment